### ISSUE TITLE
Widget is empty until I tap on it

### ISSUE THREAD

**Issue by tmarkov** [nhirokinet/bluelineconsole#277]
I've noticed that, sometimes, widgets don't fully load, and only show their contents once I tap on them. The content is there even before the tap, just invisible.

For example, a "group widget" provided by [this app](https://play.google.com/store/apps/details?id=com.makeevapps.contactswidget), shows favorite contacts that you can tap to call. If I add this widget to blue line console, I would see the widget background with no contacts shown on it. I can still call people by tapping on the right place, but their icons and names don't visibly appear. They do appear only after I tap on the widget background (outside any contacts).

This isn't the only widget I've had this problem with. Other examples would be folder widgets that contain app icons.

**Comment by nhirokinet** [comment]
Yes, that's why I keep "experimental" to the feature item at the current version (1.2.17). I planned to remove the label "(experimental)" at next version, since there are no problem report for more than an year and also did not seem to happen to me, but may be it should be reconsidered.

But unfortunately, currently I couldn't find the reason, or the way to reproduce 100%, and have no idea to do for it next.

**Comment by tmarkov** [comment]
I managed to consistently reproduce on a second phone as follows:

1. Install Blue Line Console and [Contacts Widget](https://play.google.com/store/apps/details?id=com.makeevapps.contactswidget)
2. Create a widget `-Group-` in BLC on a keyword (I've also observed the bug on homescreen, but less consistently)
3. Force stop BLC (on my phone I don't need to do that, but on the second phone the bug only appered on the first time loading the widget)
4. Open BLC
5. Enter the keyword
